routes/api.php

```diff
@@ -2,6 +2,7 @@
 
 Route::get('validations')->uses('Api\ValidationsController@view')->name('api.validations');
 Route::get('metar/{airportIcao}')->uses('Site\MetarController@get')->name('api.metar');
+Route::get('/cts/bookings')->uses('Api\CtsController@getBookings')->name('api.cts.bookings');
```

Comment:
Suggestion: I would suggest that the endpoint accepts a date range and we default to the last 7 days worth of bookings by default.